Framework/PythonInterface/inc/MantidPythonInterface/kernel/Policies/VectorToNumpy.h

```diff
@@ -87,8 +87,7 @@ template <typename ConversionPolicy> struct VectorRefToNumpy {
         typename std::remove_reference<T>::type>::type;
     // MPL compile-time check that T is a reference to a std::vector
     using type = typename boost::mpl::if_c<
-        boost::mpl::and_<std::is_reference<T>,
```

Comment:
I am not sure I understand. The class name `VectorRefToNumpy` still implies that is checks for reference? I also have not convinced myself that this doesn't break any existing behavior, will do so once this is clarified.

Reply:
This is about the return type as specified by function being exposed.